components/Firebird/Deployments/ExistingDeployment/SelectedDeployment/index.tsx

```diff
@@ -51,11 +51,13 @@ export default function SelectedDeployment({
           />
         </div>
 
-        <DeploymentBadges
-          status={deploymentEventStream.status || status}
-          chainVersion={chain.version}
-          chainId={chain.chain_id}
-        />
+        <div className="mt-1">
```

Comment:
i think you should apply this styling withing `<DeploymentBadges />`, given that all the other styles that format its div are within the component.

Reply:
Also given that this styling tag is only needed in browser and not mobile view.